docs/README.md

```diff
@@ -1,90 +1,41 @@
 
-```
-cd ..
-git clone https://github.com/grafana/grafana
-```
+### Editing the Sidebar Menu
 
-Switch your working directory to the directory this file
-(README.md) is in.
+Edit [sources/menu.yaml](sources/menu.yaml) to make changes to the sidebar. Restart the `make run` command for changes to take effect.
 
-```
-cd grafana/docs
-```
+### Adding Images
 
-An AWS config file is required to build the docs Docker image and to publish the site to AWS. If you are building locally only and do not have any AWS credentials for docs.grafana.org then create an empty file named `awsconfig` in the current directory.
+Images are currently hosted in the grafana/website repo.
 
-```
-touch awsconfig
-```
+---
 
-Then run (possibly with `sudo`):
+## Making changes live on grafana.com
 
-```
-make watch
-```
+Anytime a PR is merged to master with changes in the `docs` directory, those changes are synced via GitHub action to the grafana/website repo on the `docs-grafana` branch.
 
-This command will not return control of the shell to the user. Instead
-the command is now running a new Docker container built from the image
-we created in the previous step.
-
-Open [localhost:3004](http://localhost:3004) to view the docs.
-
-### Images and Content
-
-All markdown files are part of [this repository](https://github.com/grafana/grafana). However, all images are added to the [website repository](https://github.com/grafana/website). Therefore, the process of adding images is not as straightforward. These are the steps:
-
-1. Ensure you create a feature branch within the [website repository](https://github.com/grafana/website) to make the change. This branch needs to be based on the `old-docs` branch.
-1. Ensure the image(s) are compressed and optimised e.g. Using [tinypng](https://tinypng.com/).
-1. Add the image(s) to the `/static/img/docs` directory.
-1. Then, make a commit that adds the image(s).
-1. The Pull Request you create needs to target where you branched off, the branch `old-docs`.
-
-Finally, run:
-
-```
-make docs-build
-```
-
-This will rebuild the docs Docker image.
-
-To be able to use your image(s) you have to quit (Ctrl+C) the `make watch` command (that you run in the same directory as this README). Then simply rerun `make watch`, it will restart the docs server but now with access to your image(s).
-
-### Editing content
-
-Changes to the markdown files should automatically cause a docs rebuild and live reload should reload the page in your browser.
-
-### Troubleshooting
-
-#### Running `make watch` errors out with `Warning: Task "default" not found.`
-
-Ensure that the Docker image from the [website repository](https://github.com/grafana/website) is built using the `old-docs` branch.
+In order to make those changes live, open a PR in the website repo that merges the `docs-grafna` branch into `master`. Then follow the publishing guidelines in that repo.
```

Comment:
I don't think outside contributors can do this. Please clarify this statement. 

Also, is `docs-grafna` correct?